.env.example

```diff
@@ -118,6 +118,13 @@ OPENDAL_WASABI_BUCKET=<bucket>
 OPENDAL_WASABI_ENDPOINT=<endpoint>
 OPENDAL_WASABI_ACCESS_KEY_ID=<ak>
 OPENDAL_WASABI_SECRET_ACCESS_KEY=<sk>
+# atomicserver
+OPENDAL_ATOMICSERVER_TEST=false
```

Comment:
Those changes should not be part of this PR. Maybe you need to merge or rebase your branch with upstream?